int64 key
float32[3] accel_bias
float32[3] gyro_bias
float32[6] sigmas